lib/vyasa/sangh/comments.ex

```diff
@@ -16,7 +16,7 @@ defmodule Vyasa.Sangh.Comment do
   @primary_key {:id, Ecto.UUID, autogenerate: false}
   schema "comments" do
     field :body, :string
-    field :active, :boolean, default: true
+    field :active, :boolean, default: true #active in drafting table
```

Comment:
sync clarification needed: I don't think I have the correct mental model that aligns with the use of the word "drafting *table*" -- just need to clarify that with you when we sync

Reply:
maybe draft reflection